### PR TITLE
People Update NM

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update NM

### CODE REVIEW

File: data/nm/retired/Antonio-Maestas-2913173a-2bfe-4474-8bdc-d145f868c4de.yml [hunk 1]
```diff
@@ -8,15 +8,10 @@ image: https://www.nmlegis.gov/Images/Legislators/House/HMAES.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-11'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:nm/government
   district: '16'
```
Comment (maxknutsen) [inline data/nm/retired/Antonio-Maestas-2913173a-2bfe-4474-8bdc-d145f868c4de.yml:14]: @sroomf move to leg.

File: data/nm/legislature/Joseph-Sanchez-5d176173-a628-499c-8181-c61fc27231b8.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+id: ocd-person/5d176173-a628-499c-8181-c61fc27231b8
+name: Joseph Sanchez
+given_name: Joseph
+family_name: Sanchez
+email: Joseph.Sanchez@nmlegis.gov
+image: https://www.nmlegis.gov/Images/Legislators/House/HSAJO.jpg
+party:
+- name: Democratic
```
Comment (maxknutsen) [inline data/nm/legislature/Joseph-Sanchez-5d176173-a628-499c-8181-c61fc27231b8.yml:8]: This person should already have a file, we just need to add the new role:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:nm/government
  district: '40'
They previously served in House District 40 from 01/01/2019-12/31/2020

File: data/nm/retired/Michelle-Lujan-Grisham-cb607080-ff7b-48c8-acf4-0180e6e2f999.yml [hunk 3]
```diff
@@ -11,6 +11,10 @@ roles:
   end_date: '2023-01-01'
   type: governor
   jurisdiction: ocd-jurisdiction/country:us/state:nm/government
+offices:
+- classification: capitol
+  address: 490 Old Santa Fe Trail Room 400; Santa Fe, NM 87501
+  voice: 505-476-2200
 links:
```
Comment (maxknutsen) [inline data/nm/retired/Michelle-Lujan-Grisham-cb607080-ff7b-48c8-acf4-0180e6e2f999.yml:18]: @sroomf move out of retired, to executive maybe?

**Review by maxknutsen — COMMENTED** [review]
@sroomf changes needed in comments, thanks!

File: data/nm/retired/Michelle-Lujan-Grisham-cb607080-ff7b-48c8-acf4-0180e6e2f999.yml (hunk 3 above)
Comment (bfossen-ce) [reply data/nm/retired/Michelle-Lujan-Grisham-cb607080-ff7b-48c8-acf4-0180e6e2f999.yml:18]: We dont track executives for NM so I will keep this person in retired

**Review by maxknutsen — APPROVED** [review]

**Pull request merged**